.github/workflows/build-publish-pypi.yml

```diff
@@ -72,6 +72,8 @@ jobs:
         if: steps.check-tag.outputs.publish == 'true'
         env:
           NODE_AUTH_TOKEN: ${{secrets.npm_token}}
+          MapboxAccessTokenJupyter: ${{ secrets.mapbox_jupyter_token }}
```

Comment:
in the step "Publish kepler-jupyter to NPM":
1. add MapboxAccessTokenJupyter env
2. add `source venv/bin/activate` so proper python env will be used in `npm publish`